src/opf/ed.jl

```diff
@@ -0,0 +1,467 @@
+using SparseArrays
+
+mutable struct EconomicDispatch <: PM.AbstractPowerModel end
+
+function build_opf(::Type{EconomicDispatch}, data::Dict{String,Any}, optimizer;
+    T=Float64,
+    soft_power_balance::Bool=false,
+    soft_reserve_requirement::Bool=false,
+    soft_thermal_limit::Bool=false,
+    power_balance_penalty=350000.0,
+    reserve_shortage_penalty=110000.0,
+    thermal_penalty=150000.0,
+    iterative_ptdf::Bool=true,
+    iterative_ptdf_tol=1e-6,
+    max_ptdf_iterations=10,
+    max_ptdf_per_iteration=5,
```

Comment:
Similar thoughts on this one. With a default of 10 iterations and 5 constraints per iteration, we can add at most 50 PTDF constraints before hitting the default limits. Any instance with >50 congested lines will automatically hit a limit.

I think that 50 is a bit low.

Reply:
128 and 5 should be good?

Reply:
I like powers of `2`, so I'd bump the `5` to an `8` 😅

Note: my comments here are putting user-convenience and fool-proofing above performance. That being said, the main performance danger would be silently adding 1000s of PTDF constraints at the first iteration, which we're not doing here.